src/index.ts

```diff
@@ -53,8 +53,14 @@ export class CdktfProviderProject extends cdk.JsiiProject {
         name: `@${namespace}/provider-${providerName}`,
       },
       python: {
-        distName: `${namespace}-cdktf-provider-${providerName}`,
-        module: `${namespace}_cdktf_provider_${providerName}`,
+        distName: `${namespace}-cdktf-provider-${providerName.replace(
+          /-/gi,
+          "_"
```

Comment:
Why underscore and not hyphen?

Reply:
because the hypen is the problem and not supported in pypi as far as I can tell

Reply:
This is the original issue, where google-beta gets auto underscored by someone in the process, but only for the dist info 

```
cdktf_cdktf_provider_google-beta
cdktf_cdktf_provider_google_beta-0.0.27.dist-info
```

Reply:
Ah, I see! Python dependency management always confused me^^